### DISCUSSION TITLE
Anyone use NW as a journal?

### DISCUSSION THREAD

**Discussion by Valdus92** [vkbo/novelWriter#1434] [General]
I guess I have a habit of using thing for what they are not made and I am just curious, does anyone use Novelwriter as a journal?

The novel being- life?

**Comment by vkbo** [comment]
I do, actually. Although, recently I've been using a pen and paper instead.

**Reply by Valdus92** [reply]
Great, what I do is make every month a chapter in the "novel" and every day is a scene. I deleted plot, characters and locations. I have chapters for letters and other correspondences (like online rpg's). 

My writing pendulum also swings between analog and digital.
